app/models/ckb_sync/new_node_data_processor.rb

```diff
@@ -779,12 +779,13 @@ def build_cells_and_locks!(
       build_cell_outputs!(node_block, outputs, ckb_txs, local_block, cell_outputs_attributes, output_capacities, tags,
                           udt_address_ids, dao_address_ids, contained_udt_ids, contained_addr_ids, addrs_changes)
       if cell_outputs_attributes.present?
-        CellOutput.create! cell_outputs_attributes
+        CellOutput.create_or_find_by! cell_outputs_attributes
```

Comment:
create_or_find_by! needs to add unique index to cell_outputs table.
https://apidock.com/rails/v6.0.0/ActiveRecord/Relation/create_or_find_by